KestrelMock.Tests/MockTests.cs

```diff
@@ -32,66 +34,88 @@ public void ValidateConfiguration()
 		}
 
 		[Fact]
-		public void CanMockResponseUsingPathStartsWith()
+		public async Task CanMockResponseUsingPathStartsWith()
 		{
 			KestrelMock.Run(new ConfigurationBuilder()
 						.AddJsonFile("appsettings.json", optional: false)
 						.Build());
-			var response = HttpHelper.Get(HTTP_TEST_HOST + "starts/with/" + Guid.NewGuid());
+			var response = await HttpHelper.GetAsync(HTTP_TEST_HOST + "starts/with/" + Guid.NewGuid());
 			Assert.Contains("banana_x", response.Content);
 			Assert.Equal(200, (int)response.HttpStatusCode);
 		}
 
 		[Fact]
-		public void CanMockGetResponseUsingExactPath()
+		public async Task CanMockResponseUsingPathRegex_Matches()
+		{
+			KestrelMock.Run(new ConfigurationBuilder()
+						.AddJsonFile("appsettings.json", optional: false)
+						.Build());
+			var response = await HttpHelper.GetAsync(HTTP_TEST_HOST + "/test/1234/xyz");
+			Assert.Contains("banana_x", response.Content);
```

Comment:
We might want to use a different test result check here just to be safe. Maybe something like "regex works!" :)